app/jobs/in_person/email_reminder_job.rb

```diff
@@ -0,0 +1,56 @@
+module InPerson
```

Comment:
In rails console i'm unable to trigger the job after moving it into a new folder. I get an error that it's an uninitialized constant. How do i fix that?

Reply:
I think there's a syntax in the error and it's missing an `end` so the code never actually defines the constant (module)

Reply:
ahh ok thank you

Reply:
updated but still getting same error :(

Reply:
I added this diff locally and it worked for me
```diff
diff --git a/app/jobs/in_person/email_reminder_job.rb b/app/jobs/in_person/email_reminder_job.rb
index ac2e8169d..094bc9375 100644
--- a/app/jobs/in_person/email_reminder_job.rb
+++ b/app/jobs/in_person/email_reminder_job.rb
@@ -54,3 +54,4 @@ module InPerson
       end
     end
   end
+end
```

```
identity-idp:shannon/lg-7185-email-reminders-job-new> bundle exec rails c
Loading development environment (Rails 7.0.4)
[1] pry(main)> InPerson::EmailReminderJob
=> InPerson::EmailReminderJob
[2] pry(main)> 
```

Reply:
i was inputting the wrong thing in the console, oops!  thank you!